main.js

```diff
@@ -46,13 +44,25 @@ function displayNumbers(randomNumbers) {
                 }
             }
 
-            if (selectedOrder.length === sortedNumbers.length) {
+            if (allNumbersClicked(selectedOrder, sortedNumbers)) {
                 document.querySelector('#play_area').textContent = '';
             }
         });
     });
 }
 
+function numberHasBeenClicked(elementClass) {
```

Comment:
this function actually returns if number has NOT been clicked. either change the function name or put the bang (`!`) on the outside of the function (in the if)